### DISCUSSION TITLE
Dose tailwind provide a function to transform class name?

### DISCUSSION THREAD

**Discussion by noobSam1122** [tailwindlabs/tailwindcss#9461] [Help]
I'm working on a project that needs pop up a widget on website and it's made from tailwind. I don't want to have any conflict with existing css class name with that website, so i read the doc, it does provide a function to customize the class name ( adding custom prefix), but this will also need change the class name in HTML file.

Is there have a function to use the original class name from tailwind, but generate the prefix version class name in final HTML?

**Comment by wongjn** [comment]
I believe nothing in Tailwind can do this. You would need to construct your own build process to transform the classes in your final HTML and the selectors in your CSS.

**Reply by noobSam1122** [reply]
> I believe nothing in Tailwind can do this. You would need to construct your own build process to transform the classes in your final HTML and the selectors in your CSS.

Any hooks on build process? Checking the class name whether is tailwind might be a tough work.
